chatterbox/src/Main/index.js

```diff
@@ -0,0 +1,21 @@
+import React, { Component } from 'react';
+import logo from './logo.svg';
```

Comment:
Do SVGs work on all browsers?

Reply:
Not all browsers do but some have a basic support for it. what I did was to use .png files instead of the .SVG